.github/workflows/build-binaries.yml

```diff
@@ -87,61 +38,18 @@ jobs:
 
       - name: Package binary
         run: |
-          mkdir dungeon-revealer-linux
-          mv ./bin/dungeon-revealer-linux ./dungeon-revealer-linux/
-          mv ./node_sqlite3.node ./dungeon-revealer-linux/
-          mv ./LICENSE ./dungeon-revealer-linux/
-          mv ./README.md ./dungeon-revealer-linux/
+          mkdir dungeon-revealer-${{ matrix.os }}
```

Comment:
should be linux not ubuntu

Reply:
The runner needs it to be ubuntu for the `runs-on` statement. pkg outputs `dungeon-revealer-linux` as expected. The final packaging step on line 134 renames the zip to linux.